pkgs/development/python-modules/torch/source/default.nix

```diff
@@ -216,6 +219,7 @@ let
       rocm-comgr
       rocm-device-libs
       rocm-runtime
+      rocm-smi
```

Comment:
Where is this used? https://github.com/search?q=repo%3Apytorch%2Fpytorch%20rocm-smi&type=code

Reply:
See https://github.com/NixOS/nixpkgs/pull/431973#issuecomment-3229445898